Content.Shared/_RMC14/Holiday/RMCCustomHolidayComponent.cs

```diff
@@ -0,0 +1,13 @@
+﻿using Robust.Shared.Prototypes;
+
+namespace Content.Shared._RMC14.RMCCustomHoliday;
+
+[Prototype("customHoliday")]
```

Comment:
This is also automatically inferred from the class name
```suggestion
[Prototype]
```